README.md

```diff
@@ -10,6 +10,12 @@
 
 Lucky is still in it's early stages. A guide will be written once the API is more stabilized.
 
+## Testing
+
+You need to make sure to install the Crystal dependencies.
+1. Run `crystal deps`
+2. Run `crystal spec` from the project root.
```

Comment:
Can you change this `2` to `1`. This is a cool feature in markdown where it will correctly number them when it's rendered. This way if we add more steps and need to change the order, the number won't be out of sync

Reply:
Yeah, I noticed this in the opening section. Great idea.